flumine/markets/middleware.py

```diff
@@ -201,48 +201,20 @@ def _process_simulated_orders(self, market, market_analytics: dict) -> None:
                         k: (v.runner, v.traded.copy())
                         for k, v in market_analytics.items()
                     }
-                    live_orders_sorted = self._sort_orders(live_orders)
-                    for order in live_orders_sorted:
+                    for order in live_orders:
                         runner_traded = _lookup[(order.selection_id, order.handicap)]
                         order.simulated(market.market_book, runner_traded)
         else:  # isolation per instance
-            live_orders = list(market.blotter.live_orders)
```

Comment:
This was done for a reason, there is a gotcha here, I just can't remember what or why

Reply:
For some reason market.blotter.live_orders returns an iter of a copy of a list, and that wouldn't work with the sorting of orders but with this change it would no longer be needed